src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/AbstractSemanticTokensTests.cs

```diff
@@ -21,8 +22,8 @@ public abstract class AbstractSemanticTokensTests : AbstractLanguageServerProtoc
     {
         private protected static async Task<LSP.SemanticTokens> RunGetSemanticTokensAsync(TestLspServer testLspServer, LSP.Location caret)
         {
-            var result = await testLspServer.ExecuteRequestAsync<LSP.SemanticTokensParams, LSP.SemanticTokens>(LSP.Methods.TextDocumentSemanticTokensFullName,
-                CreateSemanticTokensParams(caret), new LSP.VSInternalClientCapabilities(), null, CancellationToken.None);
+            var result = await testLspServer.ExecuteRequestAsync<LSP.SemanticTokensParams, RoslynSemanticTokens>(LSP.Methods.TextDocumentSemanticTokensFullName,
```

Comment:
roslyn adds on to the LSP.SemanticTokens type with an extra flag, so explicitly tell streamjsonrpc to deserialize including that property.